website/language.py

```diff
@@ -72,6 +72,21 @@
 
 INVALID_EMAIL_CONFIRM_TOKEN = 'This confirmation link is invalid. Please <a href="/login/">log in</a> to continue.'
 
+CANNOT_MERGE_ACCOUNTS_SHORT = 'Cannot Merge Accounts'
+
+CANNOT_MERGE_ACCOUNTS_LONG = 'Accounts cannot be merged due to a conflict.'
```

Comment:
At the moment, the only possible conflicts are addons - but some addons prevent the merge and some don't.

If there's some specific text you'd like added there, doing so would be no problem. I need to push  another commit anyhow, to trigger the Travis build :)